docs/release-notes.txt

```diff
@@ -8,6 +8,35 @@
  Version 0.13
 **************
 
+Version 0.13.6
+==============
+
+**Release Date:** October 14, 2020
+
+**Improvements**
+
+- Agent: The ``boot_disk_source_image`` field for GCP dynamic agents and ``image_id`` field for AWS dynamic agents are now optional. They default to the latest agent images.
```

Comment:
```suggestion
- Agent: The ``boot_disk_source_image`` field for GCP dynamic agents and ``image_id`` field for AWS dynamic agents are now optional.
```
I don't think that line is necessary and it is a little confusing (it uses a hardcoded list of images and if we create a new agent image, that new image will not be used)